bin/ecompress

```diff
@@ -128,6 +128,7 @@ if [[ ${PORTAGE_COMPRESS_FLAGS+set} != "set" ]] ; then
 		# See: https://bugs.gentoo.org/672916
 		# Setting '--rm' will remove the source files after a successful compression.
 		lz4)  PORTAGE_COMPRESS_FLAGS="-m --rm";;
+		xz)   PORTAGE_COMPRESS_FLAGS="-9 -T$(__makeopts_jobs)";;
```

Comment:
I have somewhat mixed feelings about this. On one hand, we're passing `-9` to maximize compression. On the other, we're now defaulting to an arbitrary number of jobs that actually worsens compression. But I guess most people won't really care, and it has probably better gain-to-cost ratio than going for lower compression level.

Reply:
Right, people can still override it if they want.